HISTORY.md

```diff
@@ -6,6 +6,22 @@ dev
 
 -   \[Short description of non-trivial change.\]
 
+**Dependencies**
+
+- Instead of `chardet` use the MIT-licensed `charset_normalizer` for Python3
+  to remove license ambiguity for projects bundling requests. If `chardet`
+  is installed it is still going to be used instead of `charset_normalizer`
+  to keep backwards-compatibility.
+
+  You can also install chardet while installing requests by
+  specifying `[use_chardet_on_py3]` extra
+
+```bash
+    pip install "requests[use_chardet_on_py3]"
+```
```

Comment:
```suggestion
- Instead of `chardet`, use the MIT-licensed `charset_normalizer` for Python3
  to remove license ambiguity for projects bundling requests. If `chardet`
  is already installed on your machine it will be used instead of `charset_normalizer`
  to keep backwards compatibility.

  You can also install `chardet` while installing requests by
  specifying `[use_chardet_on_py3]` extra as follows:

  ```bash
  pip install "requests[use_chardet_on_py3]"
  ```
```